### ISSUE TITLE
Any interest in alphabetized lists?

### ISSUE THREAD

**Issue by jstevenco** [kiwiz/gkeepapi#91]
Hey guys,

I've forked the code and added support for alphabetizing lists (something super helpful to me for shopping etc).  I can throw a PR up if this is of interest.  Here's an example of client usage:

```
import sys, gkeepapi

def update_the_lists(label):
    print('Attempting to update list(s) with label: "' + label + '"')
    keep = gkeepapi.Keep()
    keep.login('foo@gmail.com', 'barbaz')

    targets = keep.find(labels=[keep.findLabel(label)])
    target_generator = (target for target in targets
                        if isinstance(target, gkeepapi.node.List))

    lists_processed = 0
    for target in target_generator:
        target.alphabetize()

        if target.title != '':
            print('Processing list "' + target.title + '"')
        else:
            print('Processing untitled list')
        for item in target.items:
            print(item)

        lists_processed += 1

    if lists_processed == 0:
        print('Sorry -- no lists found')

    keep.sync()

if __name__ == "__main__":
    if len(sys.argv) < 2:
        print('''Alphabetize Google Keep lists.

        Usage: python client.py [label]

        Example: python client.py "Grocery Store"

        will retrieve and alphabetize all lists with label "Grocery Store".
        ''')
    else:
        update_the_lists(sys.argv[1])
```

**Comment by kiwiz** [comment]
That sounds useful. Can you rename the method to `sort` and add an optional `key` method (for specifying an arbitrary sort)?

**Comment by jstevenco** [comment]
Sure! To clarify, you'd like to see something like 
```
target.sort(key=attrgetter('text'))
```
instead of
```
target.alphabetize()
```

**Comment by ChocoTonic** [comment]
I think he means to rename your function to something that could be integrated into the package with the option, not necessarily rewrite how your function works

**Comment by kiwiz** [comment]
Closed via #92
